src/utils/query-api.js

```diff
@@ -136,8 +136,9 @@ export class RegularExternalCommunicationDelegate
 
   async fetchUrlResponse(url: string, postData?: MixedObject) {
     this._callbacks.onBeginUrlRequest(url);
-    const requestInit =
-      postData !== undefined
+    const requestInit = url.startsWith('/')
+      ? {}
+      : postData !== undefined
```

Comment:
note that when doing a GET request, this postData should be unused anyways... yet having it be non-empty causes the GET request to still fail with CORS issues.